client/packages/requisitions/src/RnRForms/DetailView/RnRFormLine.tsx

```diff
@@ -195,6 +200,19 @@ export const RnRFormLine = ({
         textColor={textColor}
         disabled={disabled}
       />
+      <td style={{ ...readOnlyColumn, textAlign: 'center' }}>
+        {draft.lowStock !== LowStockStatus.Ok && (
+          <AlertIcon
+            double={draft.lowStock === LowStockStatus.BelowQuarter}
```

Comment:
The suggestion in the issue was to just use different colours to indicate the warning level, but I wasn't sure if confusion might arise with different themes, so thought best to stick with their current `!` or `!!` convention as well 🤷‍♀️

Reply:
thanks - I like the idea of using colours, but worried that it will be pretty unusable